src/identity/ViewingOtherProfileHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Box, Flex, Heading, Button, ButtonProps } from 'theme-ui';
+import { Link, LinkProps } from 'react-router-dom';
+import { useAuthentication } from '../authentication';
+
+const LinkButton = Button as React.FC<ButtonProps & LinkProps>;
+// TODO: the styling on this header is very ad-hoc. Let's move it into the theme.
+// TODO: this maybe doesn't belong in this module. Where does it belong?
+export const ViewingOtherProfileHeader = () => {
+  const { userId } = useAuthentication();
+  return (
+    <Box sx={{ backgroundColor: 'primary', color: 'background' }}>
+      <Flex px={3} py={2} sx={{ justifyContent: 'space-between', maxWidth: '600px' }} mx="auto">
+        <Heading as="h2" sx={{ lineHeight: 1.8 }}>
+          Patient profile
```

Comment:
Not sure about this language.  I know we've been using it, but would be weird in the passport use case.  Wondering if tt could be more like "Viewing: Steve Pole".

Reply:
In a lot of screens we show the name anyhow. Anyhow, this is feedback for Duncan.